vtexdocs/devportal
# feat: openapi desired features

#### What is the purpose of this pull request?

To update the rapidoc module to the commit containing the openapi desired features mentioned in this [PR](https://github.com/vtex/openapi-schemas/pull/936)

#### What problem is this solving?

All the problems solved are detailed in this [notion page](https://www.notion.so/vtexhandbook/OpenAPI-desired-features-2deac47764be4508af652312a47940d5)

#### How should this be manually tested?

Open the preview and check for documentation pages that could be affected by the changes.
It may be necessary to test locally, since the current openapi schemas may not have the necessary changes to test the new features, this [fork and branch](https://github.com/aliceoq/openapi-schemas/tree/feat/override-servers) contains changes to the documentations that make possible to test the new features.

#### Screenshots or example usage

Screenshots in the [notion page](https://www.notion.so/vtexhandbook/OpenAPI-desired-features-2deac47764be4508af652312a47940d5).

#### Types of changes
